tests/integration/url_generator_only/test_url_generator_get_delete.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import sys
+import os
+from odfuzz.restrictions import RestrictionsGroup
+from odfuzz.entities import DirectBuilder
+from odfuzz.fuzzer import SingleQueryable
+from pip._vendor.html5lib.constants import entities
+sys.path.append("\\Users\\I516382\\eclipse-workspace\\Jepper\\src")
+# import replacer
+print(os.environ)
+os.system("cls")
+py_version = sys.version
+metadata = open("\\Users\\I516382\\eclipse-workspace\\Jepper\\src\\metadata.xml","r").read()
```

Comment:
well.. do I really need to comment on this?- part- II.

1) for file handling, please use pathlib
https://pbpython.com/pathlib-intro.html
https://docs.python.org/3.4/library/pathlib.html

2) for "providing some metadata to test" use pytest fixtures mechanism.